spring-data-mongodb/src/main/java/org/springframework/data/mongodb/core/ReactiveMongoOperations.java

```diff
@@ -951,8 +951,11 @@ default ReactiveSessionScoped withSession(Supplier<ClientSession> sessionProvide
 	 *          {@literal null}.
 	 * @param entityClass class that determines the collection to use. Must not be {@literal null}.
 	 * @return the count of matching documents.
+	 * @since 3.4
 	 */
-	Mono<Long> count(Query query, Class<?> entityClass);
+	default Mono<Long> preciseCount(Query query, Class<?> entityClass) {
```

Comment:
How about `exactCount`? Precise implies a notion of number precision (single/double precision, decimals).